infra/nnfw/cmake/options/options_x86_64-darwin.cmake

```diff
@@ -4,3 +4,5 @@
 option(BUILD_ARMCOMPUTE "Build ARM Compute from the downloaded source" OFF)
 option(DOWNLOAD_ARMCOMPUTE "Download ARM Compute source" OFF)
 option(BUILD_XNNPACK "Build XNNPACK" OFF)
+option(DOWNLOAD_PYBIND11 "Download Pybind11 source" ON)
+option(BUILD_PYTHON_BINDING "Build python binding" ON)
```

Comment:
Wow, did you check on macOS?

Reply:
That`s my mistake

i thought that is same as linux x86-64.